src/main/kotlin/no/nav/dagpenger/regel/api/models/V2Behov.kt

```diff
@@ -0,0 +1,83 @@
+package no.nav.dagpenger.regel.api.models
+
+import de.huxhorn.sulky.ulid.ULID
+import no.nav.dagpenger.events.Packet
+import no.nav.dagpenger.regel.api.moshiInstance
+import java.time.LocalDate
+private val ulidGenerator = ULID()
+
+data class V2Behov(
+    val aktørId: String,
+    val eksternId: EksternId,
+    val beregningsDato: LocalDate,
+    val harAvtjentVerneplikt: Boolean? = null,
+    val oppfyllerKravTilFangstOgFisk: Boolean? = null,
+    val bruktInntektsPeriode: InntektsPeriode? = null,
+    val antallBarn: Int? = null,
+    val manueltGrunnlag: Int? = null,
+    val inntektsId: String? = null
+) {
+    fun toJson() = toJson(this)
+
+    companion object Mapper {
+        private val adapter = moshiInstance.adapter(V2Behov::class.java)
+
+        fun toJson(internBehov: V2Behov): String = adapter.toJson(internBehov)
+
+        fun fromJson(json: String): V2Behov? = adapter.fromJson(json)
+    }
+}
+
+data class V2InternBehov(
+    val behovId: String = ulidGenerator.nextULID(),
+    val aktørId: String,
+    val behandlingsId: BehandlingsId,
+    val beregningsDato: LocalDate,
+    val harAvtjentVerneplikt: Boolean? = null,
+    val oppfyllerKravTilFangstOgFisk: Boolean? = null,
+    val bruktInntektsPeriode: InntektsPeriode? = null,
+    val antallBarn: Int? = null,
+    val manueltGrunnlag: Int? = null,
+    val inntektsId: String? = null
+
+) {
+    fun toJson() = toJson(this)
+    fun toPacket() = toPacket(this)
+    companion object Mapper {
+        private val adapter = moshiInstance.adapter(V2InternBehov::class.java)
+
+        fun toJson(internBehov: V2InternBehov): String = adapter.toJson(internBehov)
+
+        fun fromJson(json: String): V2InternBehov? = adapter.fromJson(json)
+
+        fun toPacket(internBehov: V2InternBehov): Packet = Packet("{}").apply {
+            this.putValue(PacketKeys.BEHOV_ID, internBehov.behovId)
+            this.putValue(PacketKeys.AKTØR_ID, internBehov.aktørId)
+            when (internBehov.behandlingsId.eksternId.kontekst) {
```

Comment:
Jeg er redd for at det vil smelle nedover kjeden hvis ikke vedtak id er tilstede. Før dette går ut må "downstream" tjenestene håndtere at ikke `vedtakId` ikke er tilstede.
Er det en idé å sende EksternId på pakka i stedet?

Reply:
Ahhh, pokker. Trodde det maset stoppet ved InternBehov.
Bør bli eksternId på pakken da ja.

Reply:
Eller, er ikke behov_id den interne uulid id'en i denne sammenhengen? Er det noen av mikrotjenestene lengre ned som kobler intern-id med vedtakId?

Reply:
Mm, nei, `behov_id` blir ikke brukt med unntak av av å bruke på statussjekk (pending) og henting av behov.